integration/gethnetwork/geth_network.go

```diff
@@ -95,13 +100,20 @@ type GethNetwork struct {
 	nodesProcs       []*os.Process // The running Geth node processes.
 	logFile          *os.File
 	passwordFilePath string // The path to the file storing the password to unlock node accounts.
+	WebSocketPorts   []uint // Ports exposed by the geth nodes for
+	commPort         int
```

Comment:
Maybe `commStartPort`, and `wsStartPort` below?

Reply:
Yep 👍